### ISSUE TITLE
Support special_for_inner_method_call with Style/ClosingParenthesisIndentation

### ISSUE THREAD

**Issue by jarredholman** [rubocop-hq/rubocop#3204]
I am using the `special_for_inner_method_call` option for `Style/FirstParameterIndentation` to indent parameters to the start of the actual method call. The `Style/ClosingParenthesisIndentation` cop does not have an equivalent option and forces the closing parenthesis to be aligned with the start of the line, not the start of the method call.

---
## Expected behavior

I would like to be able to write code like this

```
expect(response).to contain_exactly(
                      { a: 1, b: 'x' },
                      { a: 2, b: 'y' }
                    )
```

even if that means adding:

```
Style/ClosingParenthesisIndentation:
  EnforcedStyle: special_for_inner_method_call
```

to rubocop.yml
## Actual behavior

The ClosingParenthesisIndentation rule doesn't seem to have any options so the only style it accepts is

```
expect(response).to contain_exactly(
                      { a: 1, b: 'x' },
                      { a: 2, b: 'y' }
)
```
## Steps to reproduce the problem

See Expected Behaviour
## RuboCop version

```
$ rubocop -V
0.39.0 (using Parser 2.3.0.7, running on ruby 2.1.5 x64-mingw32)
```

**Comment by tejasbubane** [comment]
Possibly related to #3139

**Comment by jonas054** [comment]
I think we should add a configuration option, as suggested above.
```yaml
Style/ClosingParenthesisIndentation:
  EnforcedStyle: consistent
  SupportedStyles:
    - consistent
    - special_for_inner_method_call
```
Setting `consistent` as default should work best with the default setting for `Style/FirstParameterIndentation`, I imagine.

**Comment by bbatsov** [comment]
There hasn't been much activity on this ticket and our Core Team is spread too thin on so many tasks, so I'll just close it for the sake of having a cleaner lists of tasks to focus on. We'd gladly review a PR, but it's unlikely that'd we're going to tackle this ourselves in the foreseeable future.
